### ISSUE TITLE
Error in files related to XLToolbox on loading Excel

### ISSUE THREAD

**Issue by RHWoodman** [bovender/XLToolbox#186]
Every time I load Excel, I get the following error message:

<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><logFileName>error059760_01.xml</logFileName><summary>Errors were detected in file 'C:\Users\<myusername>\AppData\Local\Temp\qi1yxnpn.kj2\XLToolbox.Legacy.XLToolboxLegacyAddin.xlam'</summary><additionalInfo><info>Excel completed file level validation and repair. Some parts of this workbook may have been repaired or discarded.</info></additionalInfo></recoveryLog>

I am also told "Excel was able to open the file by repairing or removing the unreadable content."

I am running Excel 2010, 32 bit, on a machine that is running Windows 10. The machine was originally installed with Windows 7, 64-bit and has been upgraded to Windows 8.0 --> 8.1 --> 10 (Home). My computer hardware is Intel Core i5-4430 running at 3.00 GHz. I have 12 GB of RAM, 2 GB of Video RAM, and 6 TB of hard drive storage across two physical drives.

**Comment by bovender** [comment]
That's a harmless error message. I've changed the code to load the legacy add-in (Basic code) so that certain errors experienced by some users are suppressed. I did not notice that this may cause these messages which really don't mean anything in this particular scenario.

You have two options: Revert to 7.1.2 (available for download at www.xltoolbox.net or at https://github.com/bovender/XLToolbox/releases), or keep clicking "OK" and wait for 7.1.4 which I'll release shortly, maybe today or tomorrow.

My apologies for the inconvenience!
